copyright-header.config.json

```diff
@@ -15,6 +15,8 @@
         "./extension",
         "./node_modules",
         "./package",
```

Comment:
can we remove `./package` after this?

Reply:
I left it intentionally in case anyone had previously-built package directories lying around; we can probably remove it after a few weeks when it's more likely people will have cleaned their repos